READMEs/pr-checklist.md

```diff
@@ -0,0 +1,42 @@
+# PR Checklist
```

Comment:
It might be good to incorporate this into the actual PR template: https://github.com/department-of-veterans-affairs/next-build/blob/main/.github/pull_request_template.md

Reply:
Yeah that seems like a smarter thing to do

Reply:
Removed the PR Checklist and added it to the PR template